app/views/content/funding-and-support/_hero.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="hero" style="background-color: white">
+  <img alt="Physics teacher explaining the concept of time to pupils on a whiteboard." data-lazy-disable="true" src="/packs/v1/media/images/content/hero-images/0013-3570599669a8da7d375320f4003d2d61.jpg">
+
+  <div class="hero__title">
+    <h1><span>Funding and support</span></h1>
+  </div>
+
+  <div class="hero__subtitle">
+    <p>
+      You can get support while you are training, from a range of financial support to help pay your fees to practical assistance if you are disabled.
```

Comment:
Hi @EllieNodder and @gemmadallmandfe. This hero copy feels quite difficult to read. We could simplify with something like 'You can get financial support to help fund your training, plus practical assistance if you are disabled.' Because this is top of the category page, could be worth updating today?

Reply:
agree - will update

Reply:
actually, just thinking about one of Gemma's earlier points - if we're saying 'you can get', are we implying that **everyone** is eligible for funding? (Because they aren't.) So maybe this needs rewording further.

Reply:
@GDW7 @gemmadallmandfe have now reworded this copy. Seems more in line with what we say in the other sections, too, i.e. 'we can help you to...' (/train-to-be-a-teacher and /non-uk-teachers)

Reply:
Looks good @EllieNodder